geomstats/stratified_geometry/stratified_spaces.py

```diff
@@ -4,43 +4,47 @@
 """
 
 from abc import ABC, abstractmethod
-from typing import TypeVar
 
 
-def belongs_vectorize(fun):
-    r"""Vectorize the belongs acting on Point as lists."""
+def _vectorize_point(*positions):
```

Comment:
This could go in the `geomstats/vectorization.py` file instead of being hidden in stratified_space.py?

Reply:
@ninamiolane, maybe for now we should keep in `stratified` only, because it can give the wrong impression in the context of the all library, as `vectorize` here is more "transform the input in a list".

What may be interesting is to review the decorators in `vectorize`, because I believe there's some overlap with the definition of this one and there's probably some code we can simplify there.